.openshiftio/booster.yaml

```diff
@@ -1,2 +1,7 @@
 name: WildFly Swarm - HTTP
 description: Runs a WildFly Swarm HTTP application
+versions:
```

Comment:
I think `community` and `redhat` are OK and there's a semi-agreement on those terms I think.